src/mapping/EKFSlam/EKFSlam.cpp

```diff
@@ -194,4 +194,4 @@ float EKFSlam::getValidationValue(size_t id, PointXY obstacle) {
 int EKFSlam::getNewLandmarkID() {
   counter++;
   return counter;
-}
\ No newline at end of file
+}
```

Comment:
Yeah, I was wondering what a lot of those were. As for `src/mapping`, I do believe we are using some stuff in there, like `EKFSlam.h`. Maybe we should just delete the unused files in `src/mapping`. But for the other directories I agree with deleting them completely if we aren't using anything in them.

I also agree on the `PointXY` thing - seems better to use `point_t` for that since it's Eigen. There's also some weirdness with datatypes because if I recall correctly `PointXY` uses `float` for its coordinates while other structures in our code use `double`, so it would be best to just use something consistent for that.

Reply:
Actually I think EKFSlam is the older version of the EKF (from before Abhay joined the team). Abhay's version is in the `filters` directory.

IIRC, EKFSlam isn't really working because the people who were working on it built it assuming we'd have lots of point landmarks to work with, which we don't. In the interests of cleaning up this repo I think it might be fine to delete it. (Unless one of those old team members comes back to help us integrate it with the code, it seems not worth it to dig into code that we're not familiar with.)

Reply:
Removed mapping subdirectory. Seemed like all it was currently being used for was some old test programs, and wasn't used in the main Rover program, so I removed the references to it in the `CMakeLists.txt` as well.